R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
   cat(x, ...)
 }
 
+#' Create coefficient plots from statistical model output using \pkg{ggplot2}.
+#'
+#' Create coefficient plots of \R regression output using \pkg{ggplot2}.
+#'
+#'  The \code{plotreg} function produces coefficient plots (i.e., forest plots 
+#'  applied to point estimates and confidence intervals) and works much like 
+#'  the \code{\link{screenreg}}, \code{\link{texreg}}, \code{\link{htmlreg}}, 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions. It accepts a 
+#'  single or multiple statistical models as input and internally extracts the
+#'  relevant data from the models. If confidence intervals are not defined 
+#'  in the extract method of a statistical model (see \link{extract} and 
+#'  \link{extract-methods}), the default standard errors are converted to 
+#'  confidence intervals. Most of the arguments work either like in the
+#'  \code{\link{screenreg}}, \code{\link{texreg}}, and \code{\link{htmlreg}} 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions.It is 
+#'  possible to display the plots in two ways: using the \code{type = "facet"} 
+#'  option, one forest plot applied to point estimates and confidence intervals 
+#'  will be visualised in case there is only one model. If there is more than 
+#'  one model each one will be plotted next to the other; using the 
+#'  \code{type = "forest"} option, coefficients from one or more models will 
+#'  be grouped together and displayed as a forest plot. 
+#'
+#' @param custom.title With this argument, a replacement text for the \code{ggtitle} 
+#'   that provides a title above the diagram can be provided. If an empty character 
+#'   object is provided (\code{custom.note = ""}), the note will be omitted completely.
+#' @param override.pval Replacement list of p-value vectors.
```

Comment:
It may be better to inherit this parameter from one of the other functions because the help text is already there... If that's not feasible for some reason, explain how this list needs to be structured.

Reply:
In the original `plotreg` it was named `override.pval`, I tried to make it `override.pvalues` as it is in the other functions, but I was getting an error. Hence I assumed there was a reason why you did it this way and left it the way it was. I'm going to integrate the description and explain how the line needs to be structured then.

Reply:
That's weird. What was the error?